tests/test_feedexport.py

```diff
@@ -187,6 +188,48 @@ def test_store(self):
         self.assertEqual(content, expected_content)
 
 
+class GCSFeedStorageTest(unittest.TestCase):
+
+    @mock.patch('scrapy.conf.settings',
+                new={'GCS_PROJECT_ID': 'conf_id', 'FEED_STORAGE_GCS_ACL': None }, create=True)
+    def test_parse_settings(self):
+        try:
+            from google.cloud.storage import Client
+        except ImportError:
+            raise unittest.SkipTest("GCSFeedStorage requires google-cloud-storage")
+
+        settings = {'GCS_PROJECT_ID': '123', 'FEED_STORAGE_GCS_ACL': 'publicRead' }
+        crawler = get_crawler(settings_dict=settings)
+        storage = GCSFeedStorage.from_crawler(crawler, 'gcs://mybucket/export.csv')
+        assert storage.project_id == '123'
+        assert storage.acl == 'publicRead'
+        assert storage.bucket_name == 'mybucket'
+        assert storage.blob_name == 'export.csv'
+
+    @defer.inlineCallbacks
+    def test_store(self):
```

Comment:
Would you like to add tests to cover the upload without an ACL/Policy defined (`None`)?

Reply:
Not sure.
The test will be a _copy + paste_ from `test_store` with the only difference in the assignment `acl = None`.
Probably, an option could be parameterized tests.

Reply:
In this case, you would be testing the default value for the ACL setting.